api/client/src/main/java/org/projectnessie/client/api/GetContentBuilder.java

```diff
@@ -34,6 +34,12 @@ public interface GetContentBuilder extends OnReferenceBuilder<GetContentBuilder>
 
   GetContentBuilder keys(List<ContentKey> keys);
 
+  /**
+   * Set to {@code true} to perform access checks for write/create access in addition to read access
+   * checks.
+   */
+  GetContentBuilder forWrite(boolean forWrite);
```

Comment:
In terms of backwards compatibility, shouldn't this be a default method?
```suggestion
  default GetContentBuilder forWrite(boolean forWrite) {
    return this;
  }
```